.env.example

```
@@ -408,3 +408,287 @@ NEUROLINK_DISABLE_MCP_TOOLS=false
 
 # Maximum number of tools per provider (for performance)
 NEUROLINK_MAX_TOOLS_PER_PROVIDER=5
+
+# =============================================================================
+# SYSTEM ENVIRONMENT VARIABLES (Optional - Auto-detected)
+# =============================================================================
+USER=your-username                    # System user (auto-detected)
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)